CliClient/locales/tr_TR.po

```diff
@@ -1116,19 +1145,19 @@ msgstr "Markdown bağlantısını kopyala"
 
 #, fuzzy, javascript-format
 msgid "Delete note \"%s\"?"
-msgstr "Notu sil?"
+msgstr "\"%s\ Notunu sil?"
```

Comment:
Hi @Zorbeyd, there was an error here (a missing quote after the second backslash), which made the build fail. I wonder how you've edited po file? Was it manually or did you use some tool and, if so, which one? In general it's strongly recommended not to manually edit this file, but also if you used a tool it would be good to know which one, so that I can recommend against it in the readme. Thanks!

Reply:
Hi @laurent22 . It was an manual edit. I did not have these lines in my po file, i though these fuzzy lines un-shared. I did mistake, sorry..

Reply:
@Zorbeyd, no problem, just wanted to double-check. I've already fixed the mistake so the translation will be in the next release.